tests/testament/htmlgen.nim

```diff
@@ -22,8 +28,8 @@ proc generateTestResultPanelPartial(outfile: File, testResultRow: JsonNode) =
     target = testResultRow["target"].str.htmlQuote()
     action = testResultRow["action"].str.htmlQuote()
     result = htmlQuote testResultRow["result"].str
-    expected = testResultRow["expected"].str
-    gotten = testResultRow["given"].str
+    expected = testResultRow["expected"].maybeStr
```

Comment:
Use ``getStr("")`` instead.

Reply:
Gotcha, I swear I couldn't find that when I was going trough `json.nim` heh